### PR TITLE
[2.0.1] Filebeat upgrade to 7.12.1

### PR DESCRIPTION
**Pull request by rafzei**
Can be merged after v2.0 release

### CODE REVIEW

**Review by erzetpe — DISMISSED** [review]

File: ansible/playbooks/roles/repository/files/download-requirements/requirements/x86_64/files.yml [hunk 1]
```diff
@@ -36,5 +36,5 @@ files:
   'https://charts.bitnami.com/bitnami/node-exporter-2.3.17.tgz':
     sha256: ec586fabb775a4f05510386899cf348391523c89ff5a1d4097b0592e675ade7f
 
-  'https://helm.elastic.co/helm/filebeat/filebeat-7.9.2.tgz':
+  'https://helm.elastic.co/helm/filebeat/filebeat-7.12.1.tgz':
     sha256: 5140b4c4473ca33a0af4c3f70545dcc89735c0a179d974ebc150f1f28ac229ab
```
Comment (cicharka) [inline ansible/playbooks/roles/repository/files/download-requirements/requirements/x86_64/files.yml:40]: tar has changed, how about sha256? shouldn't be different now? :)

Comment (cicharka) [inline ansible/playbooks/roles/repository/files/download-requirements/requirements/x86_64/files.yml]: tar has changed, how about sha256? shouldn't be different now? :)

**Review by cicharka — DISMISSED** [review]

**Review by to-bar — DISMISSED** [review]

**Review by przemyslavic — APPROVED** [review]

**Comment by przemyslavic** [comment]
I would just merge it. And I will test it more deeply together with OpenSearch PR.

**Review by erzetpe — APPROVED** [review]

**Pull request merged**